analysis/blacklisted/blacklisted.go

```diff
@@ -42,11 +44,20 @@ type (
 	// UrlShort is a shortened version of the URL datatype that only accounts
 	// for the IP and url (hostname)
 	UrlShort struct {
-		Url string   `bson:"host"`
-		IPs []string `bson:"ips"`
+		Url string `bson:"host"`
 	}
 )
 
+func SetBlacklistSources(res *database.Resources, result *blacklisted.Blacklist) {
```

Comment:
It is helpful to have the ability to load all of the sources which contacted a blacklisted source. Unfortunately, we cannot store this inside the blacklisted documents in MongoDB due to the size limitation on records. Here we provide the ability to load the sources at run time.